addons/analytic/models/analytic_account.py

```diff
@@ -92,6 +92,8 @@ class AccountAnalyticAccount(models.Model):
         string="Currency",
     )
 
+    plan_properties = fields.Properties('Properties', definition='plan_id.analytic_properties_definition')
```

Comment:
I really don't see how useful this can be?
Where/how can it be used?

It won't be usable with the analytic distribution widget for instance, so the only place I can think of is timesheet and/or budget, but my imagination doesn't allow me to think about a useful use case